src/hv_exc.c

```diff
@@ -91,12 +92,13 @@ static void _hv_exc_proxy(struct exc_info *ctx, uartproxy_boot_reason_t reason,
     }
 }
 
-static void hv_maybe_switch_cpu(struct exc_info *ctx, uartproxy_boot_reason_t reason, u32 type,
```

Comment:
Let's keep the new name here (after my refactoring); rendezvous isn't really what this does.